### ISSUE TITLE
Where is the conversion list?

### ISSUE THREAD

**Issue by hongquan** [danielgtaylor/arista#135]
As far as I know, the old version of Arista has a list of queuing conversion. But in version 0.9.7, it does not. Please bring it again.

**Comment by danielgtaylor** [comment]
I'm sorry, but it was removed. You couldn't really do much with the list -
it still exists but is now internal and hidden. It may return at some point
in the future, but currently I have no plans to do so.

Take care,
Daniel

2012/3/24 Nguyễn Hồng Quân <
reply@reply.github.com

> As far as I know, the old version of Arista has a list of queuing
> conversion. But in version 0.9.7, it does not. Please bring it again.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/danielgtaylor/arista/issues/135

## 

Daniel G. Taylor
http://programmer-art.org/

**Comment by hongquan** [comment]
If so, you can make it hidden by default and have a option to show it.
In case we want to convert many videos, it will be helpful to have a 
list of added conversion, to avoid duplicating when we don't remember 
if this video was added or not.

**Comment by danielgtaylor** [comment]
Okay, I'll leave this open as something to be implemented in the future. I've marked it as a spec'd feature request.

**Comment by pooky2483** [comment]
I agree, I sometimes chuck around a batch of 100 at it to convert and would like to know which one it is currently converting as I have noticed, it does not convert sequentially, e.g 01 .. 02 .. 03 .. 04 an so on. It in-fact converts randomly.
This is something I would like to see.
